packages/astro/test/units/vite-plugin-scanner/scan.test.js

```diff
@@ -17,9 +17,14 @@ describe('astro scan', () => {
 		expect(result.prerender).to.equal(false);
 	});
 
-	it('recognizes let boolean literal', async () => {
-		const result = await scan(`export let prerender = true;`, '/src/components/index.astro');
-		expect(result.prerender).to.equal(true);
+	it('throws on let boolean literal', async () => {
```

Comment:
can you add a test for `var` as well?

Reply:
Added in [b17b6dd](https://github.com/withastro/astro/pull/5297/commits/b17b6ddd00d3628b2f1c0104e90169224973f609)